rpc/config.go

```diff
@@ -43,5 +55,40 @@ type Config struct {
 
 // NewConfig builds a new config
 func NewConfig() Config {
-	return Config{Concurrency: 28, EnableTLS: false, Host: defaultRPCHost, Implementation: "grpc", RequestTimeout: 3000}
+	return Config{Concurrency: 28, EnableTLS: false, TLSVerify: true, Host: defaultRPCHost, Implementation: "grpc", RequestTimeout: 3000}
+}
+
+func (c *Config) TLSConfig() (*tls.Config, error) {
+	if !c.EnableTLS {
```

Comment:
I think, we should automatically _enable TLS_ if a root cert is provided (to minimize the number of required configuration parameters).

Let’s create a `TLSEnabled()` to encapsulate this logic

Reply:
Done